streamr-dev/streamr-platform
# [PLATFORM-1082] Edit Stream Inactivity Threshold & Partitions

Adds inactivity & partitions fields to the stream editor. Didn't update generic form styles, just used existing styles.

![image](https://user-images.githubusercontent.com/43438/67740387-970c3c80-fa50-11e9-9104-f99a100e4019.png)

![image](https://user-images.githubusercontent.com/43438/67740387-970c3c80-fa50-11e9-9104-f99a100e4019.png)

#### To Test

1. Open a stream
2. Edit partitions 
3. Edit inactivity threshold
4. State should persist after save & exit.
5. Neither should permit non-integer or blank values.
